openmetadata-service/src/main/java/org/openmetadata/service/util/EntityUtil.java

```diff
@@ -521,4 +507,12 @@ public static List<EntityReference> getEntityReferences(List<EntityReference> en
     }
     return refs;
   }
+
+  public static void validateProfileSample(Object profileSampleType, double profileSampleValue) {
```

Comment:
lets move this TableRepository as this is not common method to be in EntityUtil. Also we can just do this in JsonSchema